lib/generate/browserCollector.js

```diff
@@ -0,0 +1,191 @@
+                        // Modules are defined as being loaded 5 seconds after the
+                        // final resourceLoad event is fired. Immediately set the
+                        // loaded state to false and then set up a timer.
+                        window.modulesLoaded = false;
+                        if (window.modulesLoadedTimer) {
+                            // Delete the existing timer
+                            window.clearTimeout(window.modulesLoadedTimer);
+                        }
+                        window.modulesLoadedTimer = window.setTimeout(() => {
+                            window.modulesLoaded = true;
+                        }, 5000);
+                        // TODO: is there a queue we can check instead? This seems
+                        // too hackish to be consistent on all hardware / networks
+                        // e.g.: checkout, are we still waiting on estimate shipping
+
+                        // build up proper normalized moduleID dependency tree
+                        var moduleName = map.unnormalized
+                            ? map.prefix + '!' + map.name
+                            : map.id;
+
+                        if (excludedModules.includes(moduleName)) {
+                            return;
+                        }
+
+                        // Ignore all modules that are loaded with plugins other than text.
+                        if (
+                            moduleName.includes('!') &&
+                            !moduleName.startsWith('text!')
+                        ) {
+                            return;
+                        }
+
+                        // Ignore all modules loaded by URL
+                        if (moduleName.match(/^(https?:)?\/\//)) {
+                            return;
+                        }
+
+                        var deps = [];
+                        for (var i = 0; i < depArray.length; i++) {
+                            let depName;
+                            if (depArray[i].unnormalized) {
+                                // Plugin loading - normalise the dependency name
+                                depName =
+                                    depArray[i].prefix + '!' + depArray[i].name;
+                            } else {
+                                depName = depArray[i].id;
+                            }
+                            // Special case for domReady plugin
+                            if (depArray[i].prefix === 'domReady') {
+                                depName = 'domReady';
+                            }
+                            if (
+                                depName.includes('!') &&
+                                !depName.startsWith('text!')
+                            ) {
+                                depName = stripPlugin(depName);
+                            }
+                            if (excludedModules.includes(depName)) {
```

Comment:
We also need to exclude remote dependencies, like we've already excluded remote modules above (on line 105).

```suggestion
                            }

                            // Ignore all modules loaded by URL
                            if (depName.match(/^(https?:)?\/\//)) {
                                continue;
                            }

                            if (excludedModules.includes(depName)) {
```